GOT-StudentVersion/GOT-StudentVersion/ViewController.swift

```diff
@@ -8,18 +8,144 @@
+                
+                cell.image2.image = UIImage(named:  GOTEpisode.num4[indexPath.row].originalImageID )
+                return cell
+            }
+        case 4:
+            if let cell = tableView.dequeueReusableCell(withIdentifier: "sea1", for: indexPath) as? Sea1TableViewCell{
+                cell.nameLabel1.text = GOTEpisode.num5[indexPath.row].name
+                
+                cell.epNumber.text = "Episode: \(GOTEpisode.num5[indexPath.row].number) Season: \(GOTEpisode.num5[indexPath.row].season)"
+                
+                cell.image1.image = UIImage(named:  GOTEpisode.num5[indexPath.row].originalImageID )
+                return cell
+            }
+        case 5:
+            if let cell = tableView.dequeueReusableCell(withIdentifier: "sea2", for: indexPath) as? Sea2TableViewCell{
+                cell.nameLabel2.text = GOTEpisode.num6[indexPath.row].name
+                
+                cell.epLabel2.text = "Episode: \(GOTEpisode.num6[indexPath.row].number) Season: \(GOTEpisode.num6[indexPath.row].season)"
+                
+                cell.image2.image = UIImage(named:  GOTEpisode.num6[indexPath.row].originalImageID )
+                return cell
+            }
+        case 6:
+            if let cell = tableView.dequeueReusableCell(withIdentifier: "sea1", for: indexPath) as? Sea1TableViewCell{
+                cell.nameLabel1.text = GOTEpisode.num7[indexPath.row].name
+                
+                cell.epNumber.text = "Episode: \(GOTEpisode.num7[indexPath.row].number) Season: \(GOTEpisode.num7[indexPath.row].season)"
+                
+                cell.image1.image = UIImage(named:  GOTEpisode.num7[indexPath.row].originalImageID )
+                return cell
+            }
+            
+        default:
+            print("nice try")
+        }
+        return UITableViewCell()
+    }
+    
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return 7
+    }
+    
+    func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        let header = Functions()
+        return header.title(section)
+    }
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        tableView.dataSource = self
+        tableView.delegate = self
         // Do any additional setup after loading the view, typically from a nib.
     }
-
+    
     override func didReceiveMemoryWarning() {
         super.didReceiveMemoryWarning()
         // Dispose of any resources that can be recreated.
     }
-
-
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return 200
+    }
+    
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        guard let segueIdentifier = segue.identifier else { fatalError("No identifier in segue") }
+        
+        switch segueIdentifier {
+        case "GoTSegway" :
```

Comment:
You need to send over one episode that lives in an array of seasons. 
You can refactor this way: 
guard let indexPath = tableView.indexPathForSelectedRow,
            let detailedViewController = segue.destination as? DetailedViewController else {return}
        
        let theEpisodes = arrOfSeasons[indexPath.section][indexPath.row]
        detailedViewController.episode = theEpisodes <- this being the instance of the model that lives on the detailed VC